TRANSLATION.md

```diff
@@ -0,0 +1,77 @@
+# reactjs.org saytining O'zbek tilidagi tarjimasi
+
+Hozirgi jarayon: 
+https://github.com/reactjs/uz.reactjs.org/issues/1
+
+## Tarjima bo'yicha kelishuv
+
+| Terminning asli | Tarjim |
```

Comment:
*Tarjima

Reply:
@rustamwin you can create PR for typo fixing.